kuma/static/js/components/payments/payments-handler-utils.js

```diff
@@ -0,0 +1,23 @@
+var mdn = window.mdn || {};
+var paymentsHandlerUtils = {
+    getNewValue: function(selectedAmount) {
+        'use strict';
+        return selectedAmount < 1 || isNaN(selectedAmount)
+            ? ''
+            : '$' + selectedAmount;
+    },
+    getSelectedAmount: function(value) {
+        'use strict';
+        return value % 1 === 0 ? parseInt(value, 10) : parseFloat(value).toFixed(2);
```

Comment:
Also bothers me ;) I do not believe any math is done on this so, casting it to a `String` certainly makes sense.

Reply:
It's a pitty that this was not documented when it was written. Currently it seems to me the only real reason `parseInt` is being used, is to convert something like `2.0` to `2`.

```
2.0 % 1 === 0 ? parseInt(2.0, 10) : parseFloat(2.0).toFixed(2);
// returns 2
```

Anything else like say 2.01, will go to the `parseFloat` side and be returned as a String

```
2.01 % 1 === 0 ? parseInt(2.0, 10) : parseFloat(2.0).toFixed(2);
// returns "2.01"
```

I guess what I am saying is, why not just parse all values to a `float` and return a String? @jwhitlock Do you know why this is?

/me sure I am missing something obvious.

Reply:
I assume the reason not to use just the parseFloat() route is that then you'd end up with "$2.00" instead of "$2". But something like `parseFloat(value).toFixed(2).replace(/\.00$/, '') might do the same thing in an arguable cleaner way.

Reply:
Yup. This was a decision made that I was not aware of so, I am not going to change it for now. Will put a pin in it, and cycle back later.